.github/workflows/sync-main.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync Main Branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  check_base_status:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone base image. 
+      uses: actions/checkout@v4
+      with:
+        repository: Vanilla-OS/core-image
```

Comment:
```suggestion
        repository: Vanilla-OS/desktop-image
```

Reply:
the check is supposed to be performed on the prior image or maybe I did understand this wrong?

Reply:
Yeah, Tau informed the same to me, I accidentally commented the part when I was checking it out (resolving it).